src/Controller/CoversController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace eLife\Journal\Controller;
+
+use eLife\Patterns\ViewModel\Button;
+use eLife\Journal\ViewModel\Paragraph;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use eLife\Patterns\ViewModel\Image;
+use eLife\Patterns\ViewModel\Picture;
+use GuzzleHttp\Client;
+
+
+final class CoversController extends Controller
+{
+    public function coversAction(Request $request, string $id) : Response
+    {
+        $arguments = $this->defaultPageArguments($request);
+
+        $arguments['title'] = 'eLife Covers Landing Page';
+
+        $client = new Client(['base_uri' => 'http://prod--personalised-covers.elife.internal/personalised-covers/']);
```

Comment:
This should go through our API.

Reply:
@thewilkybarkid I don't believe there is an endpoint for this ? perhaps I'm wrong tho